apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/dao/state/phasing/PhasingPollTable.java

```diff
@@ -181,7 +181,7 @@ public long getSenderPhasedTransactionFees(long accountId, int height) throws SQ
         }
     }
 
-    public DbIterator<Transaction> getAccountPhasedTransactions(long accountId, int from, int to, int height) throws SQLException {
+    public List<Transaction> getAccountPhasedTransactions(long accountId, int from, int to, int height) throws SQLException {
```

Comment:
Connection con = null;
can be moved into try-with-resource block
`try (Connection con = ..... ) {`

Reply:
Fixed